docs/config/index.md

```diff
@@ -96,6 +96,22 @@ export default defineConfig(async ({ command, mode }) => {
 })
 ```
 
+### Environment Variables
+
+Vite doesn't load `.env` files by default. However, you can use the exported `loadEnv` helper to load the specific `.env` file needed.
```

Comment:
Maybe we could give more context?
```suggestion
Vite load `.env` files after evaluating the vite config file because the project root could be changed. However, you can use the exported `loadEnv` helper to load the specific `.env` file if needed.
```

Reply:
```suggestion
Vite doesn't load `.env` files by default as the files to load can only be determined after evaluating the Vite config, for example, the `root` and `envDir` options affects the loading behaviour. However, you can use the exported `loadEnv` helper to load the specific `.env` file if needed.
```

Maybe something like this 🤔

Reply:
Sounds good 👍